universal-application-tool-0.0.1/test/repository/WithPostgresContainer.java

```diff
@@ -25,7 +25,9 @@
 
   protected static Materializer mat;
 
-  protected static ResourceCreator resourceCreator;
+  private static ResourceCreator resourceCreator;
+
+  private static TestQuestionBank testQuestionBank = new TestQuestionBank(true);
```

Comment:
(nit) this could be a protected member. Not sure if there's much difference between using `testQuestionBank()` method or directly using `testQuestionBank` member.

Reply:
Agreed... I saw there was the protected `resourceCreator` method, so I thought I'd do the same thing for `testQuestionBank`.

Just went back and removed both "getters" and made the members `protected`.